crates/router/src/types.rs

```diff
@@ -1140,8 +746,8 @@ impl Tokenizable for CompleteAuthorizeData {
     fn set_session_token(&mut self, _token: Option<String>) {}
 }
 
-impl From<&SetupMandateRouterData> for PaymentsAuthorizeData {
-    fn from(data: &SetupMandateRouterData) -> Self {
+impl ForeignFrom<&SetupMandateRouterData> for PaymentsAuthorizeData {
```

Comment:
I see that other try from's are moved to domain models, should this be also moved?

Reply:
Since `SetupMandateRouterData` is defined in `router` crate, I couldn't move this `From` implementation to domain_models crate, Once we move all the `RouterData` aliases to domain_models crate, we can move these `From` implementation as well.